solutions/swb-reference/src/constants.ts

```diff
@@ -202,6 +202,11 @@ async function getSSMParamValue(awsService: AwsService, ssmParamName: string): P
   return response.Parameter!.Value!;
 }
 
+const SolutionId: string = 'SO0144'; //TODO: retrieve value dynamically
+const SolutionName: string = 'Service Workbench on AWS'; //TODO: retrieve value dynamically
+const SolutionVersion: string = '2.0.0'; //TODO: retrieve value dynamically
+const ApplicationType: string = 'AWS-Solutions'; //TODO: retrieve value dynamically
+
```

Comment:
Please update these names and IDs in this file as well: `deployment/solution_config`